src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+
+const InputTodo = props => {
+  const [inputText, setInputText] = useState({
+    title: "",
+  })
+
+  const onChange = event => {
+    setInputText({
+      ...inputText,
+      [event.target.name]: event.target.value,
+    })
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    if (inputText.title.trim()) {
+      props.addTodoProps(inputText.title)
+      setInputText({
+        title: "",
+      })
+    } else {
+      alert("Please write item")
```

Comment:
- Please try removing the alert method, because it is not among javascript best practices. [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

Reply:
Thanks so much @betses! I just removed the alert!